src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/JndiUserDirectoryDAO.scala

```diff
@@ -20,6 +20,10 @@ class JndiUserDirectoryDAO(providerUrl: String, user: String, password: String,
     ctx.bind(p.name, p)
   }
 
+  override def removeUser(user: RawlsUser): Future[Unit] = withContext { ctx =>
+    ctx.unbind(new Person(user).name)
```

Comment:
I kept the calls separate but I am now also calling disable.

I also added a test to show repeated enable/disable is OK